server.js

```diff
@@ -163,6 +180,169 @@ app.get("/api/episode/:id", async (request, response) => {
   }
 });
 
+// mongoDB Requests
+
+// Users
+
+app.post("/api/db/user/:id", async (request, response) => {
+  const { id } = request.params;
+  const { name } = request.body;
+  try {
+    await setUser(id, name);
+    response.status(200).send("User is set in database.");
+  } catch (error) {
+    console.error(error);
+    if (error.code === 11000) {
+      response.status(409).send("Duplicate ID");
+    }
+    response
+      .status(500)
+      .send("An unexpected error occured. Please try again later!");
+  }
+});
+
+app.get("/api/db/user/:id", async (request, response) => {
+  const { id } = request.params;
+  try {
+    const singleUser = await getSingleUser(id);
+    response.send(singleUser);
```

Comment:
Also send the status here
```suggestion
    response.status(200).send(singleUser);
```